docs/deployment/cumulus-distribution.md

```diff
@@ -20,15 +20,20 @@ To configure a deployment to use Cumulus Distribution:
  4. Rename `cumulus-template-deploy/cumulus-tf/cumulus_distribution.tf.example` to `cumulus-template-deploy/cumulus-tf/cumulus_distribution.tf`.
  5. Set the Cumulus Distribution variables in your `terraform.tfvars` (which is based on [terraform.tfvars.example](https://github.com/nasa/cumulus-template-deploy/blob/master/cumulus-tf/terraform.tfvars.example)). These include:
 
-- deploy_cumulus_distribution
-- cumulus_distribution_url
-- csdap_client_id
-- csdap_client_password
-- csdap_host_url
+- `deploy_cumulus_distribution`: Set to `true` if deploying the Cumulus Distribution API or `false` if deploying TEA.
+- `cumulus_distribution_url`: Used to override the CloudFront/API Gateway URL. This can be used if you need to insert a port for port forwarding. Not all users will need to set this and, if you do require it, it will need to be set _after_ an initial deployment. The process would be:
```

Comment:
```suggestion
- `cumulus_distribution_url`: Used to override the base URL for your distribution API that will be inserted in CMR metadata. This can be used if you need to insert a port in your distribution URL for port forwarding or specify a Cloudfront URL for distribution. Not all users will need to set this and, if you do require it, it will need to be set _after_ an initial deployment. The process would be:
```